### PR TITLE
Mention that we will rate limit SG v1-requests, part 2

### PR DESCRIPTION
**Pull request by mariadlis**

### CODE REVIEW

File: content/api/shipping-guide_2/_index.html [hunk 1]
```diff
@@ -20,5 +20,11 @@
...
   will be decomissioned on 01.07.2021. If you haven't upgraded
   to Shipping Guide 2.0 yet, we strongly
-  recommend to do that as soon as possible, and latest by 01.07.2021.<br /><br />
+  recommend to do that as soon as possible, and latest by 01.07.2021. We will 
+  progressively rate limit requests coming from version 1.0 towards this final deadline.
+  <br /><br />
+---
+
+  <h2 class="mbs">Explore our APIs</h2>
+<br /><br />
```
Comment (tomgag) [inline content/api/shipping-guide_2/_index.html:29]: I guess this is not needed

Comment (mariadlis) [reply content/api/shipping-guide_2/_index.html]: Where doe this Explore our APIs thingy come from 🤔 Anyway, better now @tomgag?

Comment (tomgag) [reply content/api/shipping-guide_2/_index.html]: Beats me. Looks fine now.

File: content/api/shipping-guide_2/_index.html [hunk 2]
```diff
@@ -20,5 +20,7 @@
   belonging to 1.0 have been phased out one by one. The last active schema, 11,
   will be decomissioned on 01.07.2021. If you haven't upgraded
   to Shipping Guide 2.0 yet, we strongly
-  recommend to do that as soon as possible, and latest by 01.07.2021.<br /><br />
+  recommend to do that as soon as possible, and latest by 01.07.2021. We will 
+  progressively rate limit requests coming from version 1.0 towards this final deadline.
```
Comment (tomgag) [inline content/api/shipping-guide_2/_index.html:24]: requests coming from version 1.0 -> incoming 1.0 requests

**Review by tomgag — APPROVED** [review]
👍

File: content/api/shipping-guide_2/_index.html (hunk 2 above)
Comment (mariadlis) [reply content/api/shipping-guide_2/_index.html:24]: @tomgag https://github.com/bring/developer-site/pull/742

**Pull request merged**